Mathlib/Topology/VectorBundle/Riemannian.lean

```diff
@@ -419,15 +421,23 @@ class RiemannianBundle where
   /-- The family of inner products on the fibers -/
   g : RiemannianMetric E
 
-/- The normal priority for an instance which always applies like this one should be 100.
-We use 80 as this is rather specialized, so we want other paths to be tried first typically. -/
-noncomputable instance (priority := 80) [h : RiemannianBundle E] (b : B) :
+/-- A fiber in a bundle satisfying the `[RiemannianBundle E]` typeclass inherits
+a `NormedAddCommGroup` structure.
+
+The normal priority for an instance which always applies like this one should be 100.
+We use 80 as this is rather specialized, so we want other paths to be tried first typically.
+As this instance is costly and quite specific, we also scope it to the `Bundle` namespace. -/
```

Comment:
I think we should point out that this instance is particularly bad because of the higher-order unification it has to perform on `E`